az_quickstart.info.yml

```diff
@@ -62,6 +62,7 @@ install:
   - chosen:chosen
   - ckeditor_bs_grid:ckeditor_bs_grid
   - coffee:coffee
+  - bootstrap_utilities:bootstrap_utilities
```

Comment:
I think this module dependency probably needs to get moved into the `az_media` module to prevent the unmet dependencies exception getting thrown during the Drupal install step on Probo (which is also causing the PHPunit tests to fail)
https://app.probo.ci/#/dashboard/build/46fe5b32-4723-4591-bfc2-6e4715ef3c40/task/5e0f50c0f6000003
```
In UnmetDependenciesException.php line 98:
                                                                               
  [Drupal\Core\Config\UnmetDependenciesException]                              
  Configuration objects provided by az_media have unmet dependencies: filter.format.az_standard 
  (bootstrap_utilities)
```

Reply:
Actually, since the basic format is provided by the install profile, the module dependency probably needs to be in both `az_quickstart.info.yml` _and_ `az_media.info.yml`...

Reply:
Oh, darn, sorry I missed this before making rapid fire commits. And thanks @joeparsons !!

I removed from `az_quickstart.info.yml` but added to `az_media.info.yml` and it's passed tests. If it works, should I leave it out of `az_quickstart.info.yml`?

Reply:
> I removed from `az_quickstart.info.yml` but added to `az_media.info.yml` and it's passed tests. If it works, should I leave it out of `az_quickstart.info.yml`?

As-is, things work because other `az_quickstart` dependencies require `az_media` which in turn requires `bootstrap_utilities` but since we're adding `bootstrap_utilities`-specific config to the basic text format inside `az_quickstart/config/install`, I think it would be better to explicitly declare `bootstrap_utilities` as a hard dependency for `az_quickstart` as well.

Reply:
Gotcha! Ok, I added it back to this file. Really appreciate you walking me through it